src/vs/workbench/contrib/mergeEditor/browser/model/diffComputer.ts

```diff
@@ -76,6 +76,10 @@ export class MergeDiffComputer implements IMergeDiffComputer {
 		}
 
 		assertFn(() => {
+			/*
+			// This does not hold (see https://github.com/microsoft/vscode-copilot/issues/10610)
```

Comment:
Can we replace this?

Reply:
There is a todo in the code. The assert is valuable for me.